tests/utils/test_async_artifacts_logging_queue.py

```diff
@@ -0,0 +1,258 @@
+import io
+import pickle
+import random
+import threading
+import time
+
+import pytest
+from PIL import Image
+
+from mlflow import MlflowException
+from mlflow.utils.async_logging.async_artifacts_logging_queue import AsyncArtifactsLoggingQueue
+
+TOTAL_ARTIFACTS = 5
+
+
+class RunArtifacts:
+    def __init__(self, throw_exception_on_artifact_number=None) -> None:
```

Comment:
nit: let's don't use partial type annotation, we can just remove `-> None` here.